versioned_sidebars/version-3.9-sidebars.json

```
@@ -197,51 +437,682 @@
+                }
+              ]
+            },
+            {
+              "type": "category",
+              "label": "カスタム値を設定",
+              "collapsible": true,
+              "items": [
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-ledger",
+                  "label": "ScalarDL Ledger のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-auditor",
+                  "label": "ScalarDL Auditor のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-schema-loader",
+                  "label": "ScalarDL Schema Loader のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-admin-for-kubernetes",
+                  "label": "Scalar Admin for Kubernetes のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-manager",
+                  "label": "Scalar Manager のカスタム値ファイルを構成"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-envoy",
+                  "label": "Scalar Envoy のカスタム値ファイルを構成"
+                }
+              ]
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-products",
+              "label": "Helm Chart を使用して Scalar 製品をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-ledger",
+              "label": "Helm Chart を使用して ScalarDL Ledger をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-auditor",
+              "label": "Helm Chart を使用して ScalarDL Auditor をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-admin-for-kubernetes",
+              "label": "Kubernetes 用の Scalar Admin をデプロイ"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/mount-files-or-volumes-on-scalar-pods",
+              "label": "ScalarDL ポッドにファイルまたはボリュームをマウント"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/use-secret-for-credentials",
+              "label": "シークレットリソースを使用"
+            }
+          ]
+        },
+        {
+          "type": "doc",
+          "id": "scalar-licensing/README",
+          "label": "製品ライセンスキーの設定"
+        },
+        {
+          "type": "doc",
+          "id": "installation-with-docker",
+          "label": "ローカル環境に ScalarDL をインストール"
```

```suggestion
          "label": "Docker を使ってローカル環境に ScalarDL をインストール"
```
Same as `sidebars.js`.